keeper/keeper.go

```diff
@@ -60,10 +60,11 @@ type Keeper struct {
 	Principal collections.Map[[]byte, math.Int]
 	Stats     collections.Item[types.Stats]
 
-	PortalOwner  collections.Item[string]
-	PortalPaused collections.Item[bool]
-	PortalPeers  collections.Map[uint16, portal.Peer]
-	PortalNonce  collections.Item[uint32]
+	PortalOwner                  collections.Item[string]
+	PortalPaused                 collections.Item[bool]
+	PortalPeers                  collections.Map[uint16, portal.Peer]
+	PortalSupportedBridgingPaths collections.Map[collections.Pair[uint16, []byte], bool]
```

Comment:
What does this composite key represent? I'm guessing the `uint16` is the chain ID but what does the slice of bytes represent?

Reply:
I may have answered my own question after reading through other parts of the code, it looks like it is the token denom? Any specific reason we use bytes instead of a string? iirc the bank module uses a similar composite key for balances but the denom uses a string.

Reply:
The second key in the pair represents the 32-byte array containing the Wormhole standardized token address on the destination chain. Which is why we can't use a string, because on EVM this would be hex encoded, but on Solana this would be base58 encoded. Hope that makes sense!